crates/store/re_entity_db/src/entity_db.rs

```diff
@@ -554,16 +554,16 @@ impl EntityDb {
             }
 
             LogMsg::ArrowMsg(_, arrow_msg) => {
-                self.last_modified_at = web_time::Instant::now();
-
+                // Convert Arrow message to chunk
                 let chunk_batch = re_sorbet::ChunkBatch::try_from(&arrow_msg.batch)
                     .map_err(re_chunk::ChunkError::from)?;
                 let mut chunk = re_chunk::Chunk::from_chunk_batch(&chunk_batch)?;
                 chunk.sort_if_unsorted();
-                self.add_chunk_with_timestamp_metadata(
-                    &Arc::new(chunk),
-                    &chunk_batch.sorbet_schema().timestamps,
-                )?
+
+                let chunk = Arc::new(chunk);
+                let timestamps = chunk_batch.sorbet_schema().timestamps.clone();
+
+                self.add_chunk_with_timestamp_metadata(&chunk, &timestamps)?
```

Comment:
I'm seeing quite a few diffs in this file and they seem spurious. Am I missing something?